src/python/CRABInterface/DataWorkflow.py

```diff
@@ -635,5 +635,26 @@ def resubmitCouchPublication(self, asourl, asodb, proxy, taskname):
                 self.logger.error(msg)
         return
 
-    def resubmitOraclePublication(self):
-        raise NotImplementedError
+    def resubmitOraclePublication(self, asourl, asodb, proxy, taskname):
+        try:
+            oracleDB = HTTPRequests(asourl.split('/crabserver')[0],
+                                    proxy,
+                                    proxy)
+        except Exception:
+            self.logger.exception('failed to connect to Oracle')
+            raise 
+
+       try:
+            data = dict()
+            data['subresource'] = 'retryPublication'
+            data['taskname'] = taskname
+
+            result = oracleDB.post('/crabserver' + asourl.split('/crabserver')[1].
```

Comment:
Hi Diego,

I think here we do not need to query again cmsweb, we can directly do the query that these 4 lines https://github.com/dmwm/CRABServer/blob/master/src/python/CRABInterface/RESTFileUserTransfers.py#L216-L219 do.

Alternatively, we can do this "post" from the client directly and modify the "crab resubmit --publication" in the client. I think it is even better since we don't pollute this class with queries that do not belong here.